dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/MainWindow.h

```diff
@@ -15,8 +15,6 @@ namespace winrt::$safeprojectname$::implementation
 
         int32_t MyProperty();
         void MyProperty(int32_t value);
```

Comment:
I think these properties should be removed. They are useless and everytime you create a new window, you're going to just get rid of it so no use.